crates/chainspec/src/spec.rs

```diff
@@ -695,7 +695,7 @@ impl From<Genesis> for ChainSpec {
             chain: genesis.config.chain_id.into(),
             genesis,
             genesis_hash: None,
-            hardforks: ChainHardforks::new(hardforks),
+            hardforks: ChainHardforks::new(ordered_hardforks),
```

Comment:
Here I'm not 100% sure but the `ordered_hardforks` collection was never read before and I have the impression that we are setting up on the few previous lines an ordering mechanism to use the `ordered_hardforks` here otherwise it has no interest right?